test/stability/testDivide/testDivide.go

```diff
@@ -259,15 +272,17 @@ func setParameters(testScenario uint) (err error) {
 	case handle:
 		// Test expects to receive 100% of packets on 0 port and 0% on 1 port
 		lessPercent = 100
+		shift = 3
 	case dhandle:
 		if testScenario != generatePart {
 			l3Rules, err = packet.GetL3ACLFromORIG("test-handle-l3rules.conf")
 		}
 		// Test expects to receive 100% of packets on 0 port and 0% on 1 port (33% total)
 		lessPercent = 100
-		passedLimit = 28 // 85 * 0.33
+		passedLimit = 28       // 85 * 0.33
+		totalPackets = 3300000 // 10 000 000 * 0.33
```

Comment:
why do we change total packets number? what if user have set total packets to 100 and now you set it to 3300000? Magics are evil!!!

Reply:
Good catch! - Thank you!